static/app/components/growingInput.tsx

```diff
@@ -50,7 +50,7 @@ export const GrowingInput = forwardRef<HTMLInputElement, InputProps>(
       if (isControlled && inputRef.current) {
         resize(inputRef.current);
       }
-    }, [props.value, isControlled]);
+    }, [props.value, props.placeholder, isControlled]);
```

Comment:
The `resize` function uses the input placeholder, so it makes sense to resize when the placeholder changes (required here since we change depending on the query value)